config/manager/controller_manager_config.yaml

```diff
@@ -1,12 +1,6 @@
-apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
-kind: ControllerManagerConfig
```

Comment:
Why are we removing `apiVersion` and `kind`? Doesn't it exists in the new version?

Reply:
We unmarshal the data ourselves, so no need for these fields anymore.